WordPress/Classes/ViewRelated/Domains/Domain registration/RegisterDomainSuggestions/RegisterDomainCoordinator.swift

```diff
@@ -0,0 +1,195 @@
+import Foundation
+
+class RegisterDomainCoordinator {
+
+    // MARK: Type Aliases
+
+    typealias DomainPurchasedCallback = ((UIViewController, String) -> Void)
+    typealias DomainAddedToCartCallback = ((UIViewController, String) -> Void)
+
+    // MARK: Variables
+
+    var site: Blog?
+    var domainPurchasedCallback: DomainPurchasedCallback?
+    var domainAddedToCartCallback: DomainAddedToCartCallback?
+    var domain: FullyQuotedDomainSuggestion?
+
+    private var webViewURLChangeObservation: NSKeyValueObservation?
+
+    init(site: Blog?, domainPurchasedCallback: RegisterDomainCoordinator.DomainPurchasedCallback? = nil) {
+        self.site = site
+        self.domainPurchasedCallback = domainPurchasedCallback
+    }
+
+    // MARK: Public Functions
+
+    func createCart(_ domain: FullyQuotedDomainSuggestion,
```

Comment:
❓ What's the difference between the passed-in `domain` param and the property `self.domain`?

Reply:
Good question!
No difference, really. It makes sense to remove this param and depend on the property for consistency.
Resolved in 42c5b70c80566e5f8c972b1580c63f0a3c89fd48